modules/travel_pay/spec/services/authenticated_session_spec.rb

```diff
@@ -53,8 +55,9 @@
       end
 
       it 'returns a cached veis_token and btsss_token' do
-        service = TravelPay::TokenService.new
-        response = service.get_tokens(user)
+        client_number = 123
+        service = TravelPay::AuthenticatedSession.new(client_number, user)
```

Comment:
🤔 What's the difference between the `AuthManager` and the `AuthenticatedSession`? Does one rely on the other? Or are they duplicative?

Reply:
A duplicate. AuthManager was supposed to be a rename to make it less... session-y. I'll fix.